.github/workflows/ci-version.yml

```diff
@@ -3,11 +3,11 @@ name: CI-version
 on:
   push:
     tags:
-      - "v*"
+      - 'v*'
```

Comment:
I would prefer using double quotes.

Reply:
sorry this was an editor auto-format, though strictly in YAML you don't need any quotes at all